processing/src/main/java/io/druid/segment/column/SimpleDictionaryEncodedColumn.java

```diff
@@ -91,6 +105,184 @@ public int getCardinality()
   }
 
   @Override
+  public HistoricalDimensionSelector makeDimensionSelector(
```

Comment:
The same as for `FilteredOffset`, this method is factored out of `QueryableIndexStorageAdapter` and tested through it